xml/System.Text.Json.Serialization/JsonConverter.xml

```diff
@@ -15,7 +15,7 @@
   <Interfaces />
   <Docs>
     <summary>Converts an object or value to or from JSON.</summary>
-    <remarks>To be added.</remarks>
+    <remarks>For more information, see <see href="~/docs/standard/serialization/system-text-json-converters-how-to.md">How to write custom converters for JSON serialization</see>.</remarks>
```

Comment:
We typically add remarks in a CDATA markdown block

```suggestion
    <remarks>
      <format type="text/markdown"><![CDATA[

## Remarks

For more information, see [How to write custom converters for JSON serialization](~/docs/standard/serialization/system-text-json-converters-how-to.md).

          ]]></format>
    </remarks>
```

Reply:
Is the rendering different with that method? If not, is there any other reason to change these now rather than waiting until we have more text to add to the remarks section? It's not just a cut-and-paste operation to move them - the links have to be manually transformed from `<see href="url">text</see>` to `[text](url)` format.